farsifonts.sh

```diff
@@ -7,25 +7,21 @@
 # You can use "axel" instead of default "wget", just install axel first and call this
 # with axel parameter.
 
-FONT_PATH="$HOME/.fonts/truetype/"
-function detect(){
-  type -P $1 >/dev/null  || { echo "Require $1 but not installed. Aborting." >&2; exit 1; }
-}
-
+FONT_PATH="$HOME/.local/share/fonts/"
 # Address of toc file, I keep this file updated.
 readonly URL="http://fzero.rubi.gd/persian-fonts-linux/list"
 readonly TOC="/tmp/list.txt"
 
-if [ $# -eq 0 ]; then
-	DOWNLOADER=wget
-	PARAMETER="-c"
-	OUTPARAM="-O"
-elif [ $1 == "wget" ]; then
-	DOWNLOADER=$1
+function detect(){
+	type -P $1 >/dev/null  || { echo "Require $1 but not installed. Aborting." >&2; exit 1; }
+}
+
+if [ $# == 0 ] || [ $1 == "wget" ]; then
+	DOWNLOADER="wget"
 	PARAMETER="-c"
 	OUTPARAM="-O"
 elif [ $1 == "axel" ]; then
-	DOWNLOADER=$1
+	DOWNLOADER="axel"
```

Comment:
I prefer something that is available.  in every distro by default. you can make it customizable , but I am against changing the default